app/views/search/_form.html.haml

```diff
@@ -1,6 +1,13 @@
 - input_class = large_inputs ? 'input-lg' : ''
 - btn_class = large_inputs ? 'btn-lg' : ''
 
+%p Écrivez puis sélectionnez dans la liste :
```

Comment:
These instructions are pretty long and most of it is implicit. Could be replaced by a line in FAQ or a help popup for less pollution in main page.

Reply:
I think too.
It was suggested by https://github.com/djakk in his/her PR https://github.com/covoiturage-libre/covoiturage-libre/pull/195
We have to decide with others.

Reply:
@AlexBillbao , @djakk, @maxb29, @ndelavergne ? What's your opinion about these instructions on the main page ?

Reply:
@webplate @nicolasleger I like when it is verbose :) but maybe the red error message "sélectionnez une ville *dans* la liste" is enough

Reply:
Sinon maxb29 propose : 
"Écrivez puis sélectionnez dans la liste:
Une ville de départ et/ou une ville d'arrivée"